__tests__/integration.test.js

```diff
@@ -74,4 +74,34 @@ describe('/api/articles/:article_id', () => {
         })
     })
  })
- 
\ No newline at end of file
+describe('/api/articles', () => {
+    test('GET: 200, returns an articles array of article objects with a comments count added and body removed', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const { articles } = body;
+            articles.forEach((article) => {
+                expect(article).toHaveProperty('comment_count')
+                expect(article).not.toHaveProperty('body')
+                expect(article).toHaveProperty('article_id');
+                expect(article).toHaveProperty('author');
+                expect(article).toHaveProperty('title');
+                expect(article).toHaveProperty('topic');
+                expect(article).toHaveProperty('created_at');
+                expect(article).toHaveProperty('votes');
+                expect(article).toHaveProperty('article_img_url');
+            })
```

Comment:
The test could be more robust
- If the array is empty the `forEach` won't run and you'll get a default test pass
- `.toHaveProperty` is only asserting that a property exists but doesn't say anything about the value on that property.

Take _article_id_ for example:
`expect(typeof article.article_id).toBe("number")` 
This assertion tells us two things
- The property _article_id_ exists, otherwise it wouldn't be a number
- _and_ that the value must be some value of number

So we can get more confidence from our test while using the same number of code lines